src/main/java/net/snowflake/client/core/DataConversionContext.java

```diff
@@ -12,25 +12,54 @@
  * to a single result set. a.k.a each result set object should have its own formatter info
  */
 public interface DataConversionContext {
-  /** timestamp_ltz formatter */
+  /**
+   * timestamp_ltz formatter
```

Comment:
the existing comments should be used in `@return` - we are returning specific formatters for mentioned types (here and in methods below)